src/tests/unit/tests/common/components/cards/simple-card-row.test.tsx

```diff
@@ -22,13 +22,25 @@ describe('SimpleCardRow', () => {
     });
 
     it('renders', () => {
-        const testSubject = shallow(<SimpleCardRow {...props} />);
-        expect(testSubject.getElement()).toMatchSnapshot();
+        const renderResult = render(
+            <table>
```

Comment:
Why is the tested component wrapped in additional markup here? Doesn't the snapshot work testing just the `SimpleCardRow`?

Reply:
We did this because we were getting
**Warning: validateDOMNesting(...): <tr> cannot appear as a child of <div>** issue. To resolve that, we needed to add additional markup tags.

Reply:
We did this because we were getting
**_`Warning: validateDOMNesting(...): <tr> cannot appear as a child of <div>`_**  issue. To resolve this, we needed to add additional markup tags.